src/metabolic_ninja/schemas.py

```diff
@@ -48,6 +48,7 @@ class Meta:
     model_id = fields.Integer(required=True)
     product_name = fields.String(required=True)
     max_predictions = fields.Integer(required=True)
+    aerobic = fields.Boolean(required=False)
```

Comment:
The field should be presesnt (but allowed to be null), so I think we want `required=True, allow_none=True` here